### ISSUE TITLE
No Async versions of IApiOutputCache

### ISSUE THREAD

**Issue by gorgi** [filipw/AspNetWebApi-OutputCache#158]
I couldn't find in previous issues whether this was considered, but I think its a good option to have. For example, if Redis is used as caching service, all requests to Redis can be made as async. It is supported by existing C# Redis clients.

**Comment by filipw** [comment]
Yeah it's a good idea; I think I will break the interface to remove the obsolete member https://github.com/filipw/AspNetWebApi-OutputCache/blob/dev/src/WebApi.OutputCache.Core/Cache/IApiOutputCache.cs#L12 and will add async versions at the same time.

**Comment by leoddn** [comment]
Hello Filip,

Is an Async version of `IApiOutputCache` still on the way ? It would be very useful for my project.

Thank you !

**Comment by TimGebhardt** [comment]
FYI the API for output caching was recently updated to support async providers: https://blogs.msdn.microsoft.com/webdev/2016/12/05/introducing-the-asp-net-async-outputcache-module/

**Comment by johnib** [comment]
Any update on this? we'd like to have such interface as well
